guides/wordpress-openid-connect-client-plugin-integration/index.md

```diff
@@ -34,7 +26,21 @@ You can also visit the plugin’s page to download, install and activate manuall
 
 [wordpress.org/plugins/miniorange-openid-connect-client/](https://wordpress.org/plugins/miniorange-openid-connect-client/)
 
-## Step 3. Configure the plugin
+### Setup the App on did
+
+You will need a DID account. [Sign up]({{ site.auth_origin }}) to create one now.
+
+After signing up, you will be directed to set up your first app.
+
+Give your new app a descriptive name e.g. ‘Wordpress Demo’ and use the full URL of the website in the ‘host’ box.
+
+Leave the ‘Test mode’ box unchecked to use this app on a live Wordpress website. Test mode can be used for localhost installations.
```

Comment:
Test mode isn't just for localhost. If you have a staging or example site on the internet you can use Test their too

Reply:
ah coool ok, I'll leave this line out.  Perhaps somewhere else we can discuss what the test mode is for and how it works.